libs/dh/core/shell/src/lib/dh-primary-navigation.component.html

```diff
@@ -32,13 +32,19 @@
         transloco("marketParticipantGridAreas")
       }}</watt-nav-list-item>
     </ng-container>
-    <ng-container *dhFeatureFlag="'start_wholesale_process_feature_flag'">
+  </watt-nav-list>
+
+  <ng-container *dhFeatureFlag="'start_wholesale_process_feature_flag'">
+    <watt-nav-list
+      [expandable]="true"
+      [title]="transloco('wholesale.menuGroupLabel')"
+    >
       <watt-nav-list-item link="/wholesale/start-process">{{
-        transloco("wholesaleStartProcess")
+        transloco("wholesale.startProcess")
       }}</watt-nav-list-item>
       <watt-nav-list-item link="/wholesale/overview">{{
-        transloco("wholesaleOverview")
+        transloco("wholesale.overview")
```

Comment:
Btw, would you mind hiding or removing the overview page for us as it is not currently used?

Reply:
I've removed it only from the sidebar nav for now. Can help will a bigger refactor (removing the related code) if necessary in the future.